inference-engine/ie_bridges/java/samples/face_detection_sample_async/Main.java

```diff
@@ -0,0 +1,259 @@
+import org.opencv.core.*;
+import org.opencv.imgcodecs.*;
+import org.opencv.videoio.*;
+import org.opencv.videoio.VideoWriter;
+import org.opencv.imgproc.Imgproc;
+
+import java.util.LinkedList;
+import java.util.Vector;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.TimeUnit;
+import java.util.Map;
+import java.util.Queue;
+import java.util.ArrayList;
+import java.util.HashMap;
```

Comment:
Can we enable code style check for Java?

Reply:
I think so. Do you mind if we will make it on next iteration?

Reply:
Can I do it later? We want to merge this PR before the feature freeze.

Reply:
Sure, thanks